index.bs

```diff
@@ -1520,94 +1459,189 @@ If the attestation root certificate is not dedicated to a single WebAuthn Authen
 should be specified in the attestation certificate itself, so that it can be verified against the <a>authenticatorData</a>.
 
 
+# [RP] Operations # {#rp-operations}
+
+Upon successful execution of a {{makeCredential()}} or {{getAssertion()}} call, the [RP]'s script receives a
+{{ScopedCredentialInfo}} or {{AuthenticationAssertion}} structure respectively from the client. It must then deliver the
+contents of this structure to the <a>[RP]</a>, using methods outside the scope of this specification. This section describes the
+operations that the [RP] must perform upon receipt of these structures.
+
```

Comment:
Isn't that too limiting? It's fine to break up the structure into pieces if desired, each of the elements stands on its own.